.github/workflows/BuildPR.yml

```diff
@@ -0,0 +1,41 @@
+name: pull request
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        xcode: [ Xcode_14.1.0, Xcode_13.3 ]
+        os: [ macos-12 ]
+        abi: [ x86_64 ]
```

Comment:
Nit: This isn't the ABI, this is the architecture (the ABI would e.g. be `gnu` in `x86_64-unknown-linux-gnu`, on macOS there's only one standard ABI though, so it's usually omitted). See [the Clang docs](https://clang.llvm.org/docs/CrossCompilation.html#target-triple) for details.

Reply:
Ahh, this is copy&paste leftover from other repo. 
Here I remove it https://github.com/SwiftGit2/SwiftGit2/pull/206

Reply:
ok, the name is misleading

Reply:
https://github.com/SwiftGit2/SwiftGit2/pull/207